woocommerce/woocommerce
# fix: label improvement on my order pages

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/43627

Adding an `aria-label` for screen readers to the link with the order number.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

- As a merchant, ensure you allow customers to create an account by enabling the "Allow customers to create an account during checkout" setting on **WooCommerce > Settings > Accounts & Privacy**
- As a customer, place an order & create the account on checkout
- As a customer, navigate to the "My account > Orders" page
- The `aria-label` attribute should be added to the link with the order number
<img width="810" alt="Screenshot 2024-06-11 at 1 42 26 PM" src="https://github.com/woocommerce/woocommerce/assets/273592/d6d3fc56-509f-47c1-aa27-7ec2327d167a">



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
fix: label improvement on my order page template
#### Comment
</details>
